src/status_im/subs/profile.cljs

```diff
@@ -126,9 +125,9 @@
 
 (re-frame/reg-sub
  :multiaccount/default-account
- :<- [:profile/wallet-accounts]
+ :<- [:wallet/accounts]
  (fn [accounts]
-   (wallet.utils/get-default-account accounts)))
```

Comment:
Should we move the methods `get-default-account` and `default-address` method from `legacy.status-im.wallet.utils` to `status-im.wallet.utils` ns ?

Reply:
We can just add it in a separate pr, there's enough noise in this one 😰